### PR TITLE
Tier 2 Template Review

### PR DESCRIPTION
**Pull request by natalialuzuriaga**
## Problem

Due to updates made to the maturity model from outbounding DedupliFHIR, the Tier 2 templates need to be reviewed to ensure they align with newest changes.

## Solution

- Added missing Shipping Releases section
- Commented out Recommended sections (PR, Team Guidelines, Documentation)
- Added missing comments and cleaned up templates

### CODE REVIEW

**Comment by natalialuzuriaga** [comment]
Questions:

-In the spreadsheet, there is a section called. "Reviewing PRs". This section is named as "Code Review" in this template. "Code Review" or "Reviewing PRs"?

File: tier2/{{cookiecutter.project_slug}}/CONTRIBUTING.md [hunk 1]
```diff
@@ -12,35 +15,46 @@ We encourage you to read this project's CONTRIBUTING policy (you are here), its
...
 
 1.  Fork the project 
-1.  Check out the `main` branch 
-1.  Create a feature branch
-1.  Write code and tests for your change 
-1.  From your branch, make a pull request against `cmsgov/cmsgov-example-repo/main` 
-1.  Work with repo maintainers to get your change reviewed 
-1.  Wait for your change to be pulled into `cmsgov/cmsgov-example-repo/main`
-1.  Delete your feature branch
+2.  Check out the `main` branch
```
Comment (decause-gov) [inline tier2/{{cookiecutter.project_slug}}/CONTRIBUTING.md:41]: using `1.` will (should?) auto-number elements in a markdown list, but I'm ok with explicit numbering too :)

File: tier2/{{cookiecutter.project_slug}}/README.md [hunk 2]
```diff
@@ -11,38 +11,50 @@
...
 ## Core Team
 
 An up-to-date list of core team members can be found in [MAINTAINERS.md](MAINTAINERS.md). At this time, the project is still building the core team and defining roles and responsibilities. We are eagerly seeking individuals who would like to join the community and help us define and fill these roles.
 
-<!--
+<!-- 
 ## Documentation Index 
-**{list of .md at top directory and descriptions}**
 
+TODO: This is a like a 'table of contents" for your documentation. Tier 0/1 projects with simple README.md files without many sections may or may not need this, but it is still extremely helpful to provide "bookmark" or "anchor" links to specific sections of your file to be referenced in tickets, docs, or other communication channels.
```
Comment (decause-gov) [inline tier2/{{cookiecutter.project_slug}}/README.md:26]: +1 on the description here!

**Review by decause-gov — APPROVED** [review]
Great updates Nat! If we haven't made these tweaks in other tiers, may be worthwhile.

LGTM +1 🚢

**Pull request merged**